mautic/mautic
# Form submission batch delete fixed

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | true
| New feature? | false
| Related user documentation PR URL | null
| Related developer documentation PR URL | null
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/3091
| BC breaks? | false
| Deprecations? | false

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
The batch delete button for form results was still among the top right action buttons and it did not work. This PR moves it into the table and fix the functionality using the standard controller method to keep the code DRY.

#### Steps to test this PR:
1. Apply this PR
2. Clear the cache
3. Repeat the test
- Note that the batch delete button is now consistent with the rest of the table UI.
- The batch delete and normal delete should work as expected.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Go to a form results where you have some submissions already and try to batch-delete at least one row. You should get an error.
2. Notice the delete button is in the top right hand corner.
